apps/right-to-rent-check/acceptance/features/check-not-needed-date.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Given I am on /check-not-needed-date');
+
+Before((
+  I,
+  checkNotNeededDatePage
+) => {
+  I.visitPage(checkNotNeededDatePage, steps);
+});
+
+Scenario('When I click on the back to guidance link I am taken to https://www.gov.uk/check-tenant-right-to-rent-documents', (
+  I
+) => {
+  I.click('#content a');
+  I.seeInCurrentUrl('www.gov.uk/check-tenant-right-to-rent-documents');
```

Comment:
You should use this https://github.com/Codeception/CodeceptJS/blob/master/docs/helpers/WebDriverIO.md#seecurrenturlequals

Reply:
Otherwise, this is is good